soccer/gameplay/plays/offense/adaptive_formation.py

```diff
@@ -284,17 +285,20 @@ def on_exit_dribbling(self):
         self.remove_subbehavior('dribble')
 
     def on_enter_shooting(self):
-        # TODO: Use moving kick when completed
         self.kick = skills.pivot_kick.PivotKick()
+
+        # Same params as basic_122
+        self.kick.aim_params['error_threshold'] = 0.3
```

Comment:
It would be nice to have these as a constant somewhere so they can be shared. You could even have a constant dict and update the dict here. This will probably surprise people in the future if there's multiple copies :)

Reply:
You could easily have 2 big ones, shooting and passing inside constants or something (or pivot kick). That'll make tuning all the specific plays really easy if you just want to make shots looser or something